lib/services/identity/controllers/identity-list-controller.js

```diff
@@ -36,13 +38,10 @@ class IdentityListController {
 		payload.type = type;
 		let channelId = (req.identity.channel || {}).id;
 
-		// only allow channel from the payload from an admin
+		// TODO should none admin audience be able to post a new channel?
```

Comment:
I'm not sure if this is supposed to be this way.  But it looks like any audience clam can create a new channel.